k8s/src/main/scala/io/buoyant/k8s/ServiceNamer.scala

```diff
@@ -5,10 +5,7 @@ import com.twitter.finagle.{Service => _, _}
 import com.twitter.finagle.service.Backoff
 import com.twitter.finagle.util.DefaultTimer
 import com.twitter.util._
-import io.buoyant.k8s.Ns.ObjectCache
 import io.buoyant.k8s.v1._
-import java.net.InetSocketAddress
-import scala.collection.mutable
 
 /**
  * Accepts names in the form:
```

Comment:
I think this class needs to be refactored to be very similar to the EndpointsNamer.  That is, it should use `NsApi.service(serviceName).activity` instead of ServiceCache.  Ideally, ServiceCache can be deleted entirely.